src/resources/i18n/translations/en.json

```
@@ -212,43 +212,42 @@
 	"Auto": "Auto",
 	"Autoscroll Off": "Auto-scroll off",
 	"Autoscroll On": "Auto-scroll on",
-	"Back To App": "Back to app",
-	"Back To Login": "Back To login",
+	"Back To App": "Back To App",
+	"Back To Login": "Back To Login",
 	"Back": "Back",
 	"Balance": "Balance",
-	"Bitcoin Wallet": "Bitcoin wallet",
+	"Bitcoin Wallet": "Bitcoin Wallet",
 	"Boost Amount for this Podcast": "Boost amount for this podcast",
 	"Boost Amount": "Boost amount",
 	"Boost Pay Error": "Boost pay error",
 	"Boost Pay Issues": "Payment issues",
 	"Boost Sent": "Boost sent",
 	"Boost splits": "Boost splits",
 	"Boost": "Boost",
-	"Bug Report": "Bugreport",
+	"Bug Report": "Bug Report",
```

This yields the likes of "Bug reports for service", as a personal pet peeve.
```suggestion
	"Bug Report": "Report a Bug",
```

---

👍